### PR TITLE
Chges doc

### PR DESCRIPTION
**Pull request by lcontami**
Just some typo corrections / suggestions for changes in the ECPY documentation (developer guide)

### CODE REVIEW

File: docs/source/dev_guide/measure.rst [hunk 1]
```diff
@@ -44,13 +44,13 @@ pre-hook can have two purposes :
 
 Adding a pre-hook requires to :
 
-- implement the logic by subclassing |BasePreHook|. The methods that can be
+- implement the logic by subclassing |BasePreexecutionHook|. The methods that can be
```
Comment (MatthieuDartiailh) [inline docs/source/dev_guide/measure.rst]: This looks weird are you sure ?

File: docs/source/dev_guide/measure.rst [hunk 2]
```diff
@@ -119,21 +119,19 @@ asked not to run them). They are hence perfectly fitted to run clean up.
...
     executed by passing to the active engine. The post hook can inspect the
     measure it belongs to to identify whether the execution finished correctly
     ('task_execution_result' member).
-  - pause/resume/stop: to implement is the run method execution can take a
+  - pause/resume/stop: to implement if the run method execution can take a
     long time (typically if tasks are involved).
   - list_runtimes: let the measure know the runtime dependencies (such as
     instrument drivers) if any.
 
-  Additionally if any entry is contributed to the task hierarchy they should
```
Comment (MatthieuDartiailh) [inline docs/source/dev_guide/measure.rst]: Why were those removed ?

File: docs/source/substitutions.sub [hunk 3]
```diff
@@ -79,15 +79,15 @@
 
 .. |BaseToolDeclaration| replace:: :py:class:`~ecpy.measure.base_tool.BaseToolDeclaration`
 
-.. |BasePreHook| replace:: :py:class:`~ecpy.measure.hooks.base_hooks.BasePreHook`
+.. |BasePreExecutionHook| replace:: :py:class:`~ecpy.measure.hooks.base_hooks.BasePreExecutionHook`
```
Comment (MatthieuDartiailh) [inline docs/source/substitutions.sub]: This confirms my first comment. Could you try to build the docs using 'make html' from the docs directory. You need sphinx installed. To see if there are any errors.

File: docs/source/user_guide/measure_edition.rst [hunk 4]
```diff
@@ -124,21 +124,21 @@ By default a single pre-hook is attached to the measure: the one responsible
 for running the tests of the measure, it cannot be removed.
 
 As usual you can add new tools using the add button and edit them when they are
-selected. The use of pre/post hooks being pretty straightforward it will not be
-detailed here.
+selected. The use of pre/post hooks being pretty straightforward it will not be 
+detailed here.
```
Comment (MatthieuDartiailh) [inline docs/source/user_guide/measure_edition.rst]: Trailing white space you need to set up your editor to avoid those.

File: docs/source/user_guide/measure_edition.rst [hunk 5]
```diff
@@ -124,21 +124,21 @@ By default a single pre-hook is attached to the measure: the one responsible
...
-please note that this kind of notification can be time consuming and hence it is
+monitor (and hence what it is not supposed to do). A monitor is supposed to ask for
+notifications when some entries are updated in the database and react to that
+change in way that lets the user know what is currently going on. First please 
+note that this kind of notification can be time consuming and hence it is 
 better not to observe values inside tight loops (whose each iteration is around
 30 ms). Second a monitor should strive for stability and low memory consumption
-so that the measure does not crash because of it, which is why it should not
-try to plot all the data acquired by the measure but leave this work to
+so that the measure does not crash because of it, which is why it should not
```
Comment (MatthieuDartiailh) [inline docs/source/user_guide/measure_edition.rst]: same here

File: docs/source/user_guide/measure_edition.rst [hunk 6]
```diff
@@ -124,21 +124,21 @@ By default a single pre-hook is attached to the measure: the one responsible
...
-so that the measure does not crash because of it, which is why it should not
-try to plot all the data acquired by the measure but leave this work to
+so that the measure does not crash because of it, which is why it should not 
+try to plot all the data acquired by the measure but leave this work to 
 external programs.
 
 Ecpy comes with a built-in monitor which can display the values of the database
-entries. It can perform some minimal formatting on those entries and you can
-build new ones with custom formatting. It attached by default to all measures.
+entries. It can perform some minimal formatting on those entries and you can
```
Comment (MatthieuDartiailh) [inline docs/source/user_guide/measure_edition.rst]: Could you add back the removed sentence ?

File: docs/source/substitutions.sub (hunk 3 above)
Comment (lcontami) [inline docs/source/substitutions.sub]: The doc builds without error
